website/yarn-error.log

```diff
@@ -0,0 +1,6720 @@
+Arguments: 
```

Comment:
maybe this should be added to .gitignore?

Reply:
Good call